src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -76,7 +76,7 @@ Napi::Value CoreWrap::read_model_sync(const Napi::CallbackInfo& info) {
 Napi::Value CoreWrap::read_model_async(const Napi::CallbackInfo& info) {
     try {
         ReadModelArgs* args = new ReadModelArgs(info);
```

Comment:
@t-jankowski you told that it's possible to don't use asterisk, but if I change like in code below, it doesn't work
```suggestion
        ReadModelArgs args = ReadModelArgs(info);
```

and replace:
```
ReaderWorker* _readerWorker = new ReaderWorker(info.Env(), _core, &args);
```

Reply:
It appears that `ReaderWorker` object destruction is handled by `napi` - see `AsyncWorker::Destroy()`. Note that it wouldn't be destructed if `AsyncWorker::SuppressDestruct()` is called on an object.